Slack.NetStandard/WebApi/AdminAnalyticsApi.cs

```diff
@@ -21,7 +21,7 @@ public Task<HttpResponseMessage> GetFile(string type, DateTime? date = null)
             };
             if (date.HasValue)
             {
-                dict.Add("date", date.Value.ToUniversalTime().ToString("yyyy-MM-dd"));
+                dict.Add("date", date.Value.ToString("yyyy-MM-dd"));
```

Comment:
I think this probably shouldn't be doing a ToUniversalTime(),in my timezone this unit test failed as it changed the date to the day prior.

I would think it's best to assume the user should always provide a UTC date stamp? Alternatively we can not change this Api and just change the unit test to specify a UTC Time Kind (which i've also done)

Reply:
For now can we change the test please - that way this PR is kept stars related and it's easier to track in the future